tests/core/utilities/test_event_filter_builder.py

```diff
@@ -1,15 +1,14 @@
 import pytest
 
-from hypothesis import (
-    given,
-    strategies as st,
-)
-
 from web3._utils.events import (
     DataArgumentFilter,
     TopicArgumentFilter,
     normalize_topic_list,
 )
+from web3.tools.hypothesis import (
```

Comment:
This import looks broken.

Reply:
Broken in the sense it looks off/should be done a different way ? or non functional ? all tests passing

Reply:
Ignore this one... commented on the wrong thing.